beetmoverscript/constants.py

```diff
@@ -34,3 +34,9 @@
     'pub/mobile/release': 'release',
     'pub/firefox/release': 'release',
 }
+RELEASE_BRANCHES = (
+    'mozilla-central',
+    'mozilla-aurora',
+    'mozilla-beta',
+    'mozilla-release'
```

Comment:
Maybe add in mozilla-esr52?
We'll need to update docs to update this per new esr branch.